vnext/Desktop.UnitTests/React.Windows.Desktop.UnitTests.vcxproj

```diff
@@ -60,7 +60,6 @@
         WIN32;
         BOOST_ASIO_HAS_IOCP;
         FOLLY_NO_CONFIG;
-        GTEST_LANG_CXX11=1;
```

Comment:
>GTEST_LANG_CXX11=1; [](start = 8, length = 19)

I am just curious what this definition was about: is it to enable C++11? Can we use C++11 after its removal?

Reply:
Yes, we can. My guess is this flag was required for older versions of GTest, that didn't allow C++0x by default.
This is no longer the case.

MOREOVER, this flag didn't at all belong to this specific VCXPROJ, which is based on MSUT, and not GTest.